integration/container/stop_windows_test.go

```diff
@@ -1,68 +0,0 @@
-package container // import "github.com/docker/docker/integration/container"
-
-import (
-	"context"
-	"strconv"
-	"testing"
-	"time"
-
-	containertypes "github.com/docker/docker/api/types/container"
-	"github.com/docker/docker/integration/internal/container"
-	"gotest.tools/v3/assert"
-	"gotest.tools/v3/poll"
-	"gotest.tools/v3/skip"
-)
-
-// TestStopContainerWithTimeout checks that ContainerStop with
-// a timeout works as documented, i.e. in case of negative timeout
-// waiting is not limited (issue #35311).
-func TestStopContainerWithTimeout(t *testing.T) {
-	skip.If(t, testEnv.OSType == "windows")
-	defer setupTest(t)()
-	client := testEnv.APIClient()
-	ctx := context.Background()
-
-	testCmd := container.WithCmd("sh", "-c", "sleep 2 && exit 42")
-	testData := []struct {
-		doc              string
-		timeout          int
-		expectedExitCode int
-	}{
-		// In case container is forcefully killed, 137 is returned,
-		// otherwise the exit code from the above script
-		{
-			"zero timeout: expect forceful container kill",
-			1, 0x40010004,
```

Comment:
Were these exit-codes different between Linux and Windows? (And is that what's causing the tests to fail?).


```
=== FAIL: github.com/docker/docker/integration/container TestStopContainerWithTimeout/zero_timeout:_expect_forceful_container_kill (5.20s)
    stop_test.go:106: assertion failed: 3221225786 (inspect.State.ExitCode int) != 1073807364 (tc.expectedExitCode int)

=== FAIL: github.com/docker/docker/integration/container TestStopContainerWithTimeout/unlimited_timeout:_expect_graceful_container_stop (6.11s)
    stop_test.go:106: assertion failed: 3221225786 (inspect.State.ExitCode int) != 0 (tc.expectedExitCode int)

=== FAIL: github.com/docker/docker/integration/container TestStopContainerWithTimeout/too_small_timeout:_expect_forceful_container_kill (5.15s)
    stop_test.go:106: assertion failed: 3221225786 (inspect.State.ExitCode int) != 1073807364 (tc.expectedExitCode int)

=== FAIL: github.com/docker/docker/integration/container TestStopContainerWithTimeout/big_enough_timeout:_expect_graceful_container_stop (6.27s)
    stop_test.go:106: assertion failed: 3221225786 (inspect.State.ExitCode int) != 0 (tc.expectedExitCode int)

=== FAIL: github.com/docker/docker/integration/container TestStopContainerWithTimeout (0.04s)

```

Reply:
oh, but you created a special case for windows to set a different code. Hm.. no idea then 😂

Reply:
The exict code is different that the expected code from the previous tests that weren't run. It's possible that the exitcode 3221225786 is the correct one here, but there's still the case of the non-zero exit code when the container was supposed to exit cleanly.
I didn't spend time investigating this deeper yet, because this was just something I encountered randomly and thought it would be a simple fix 😂 
I'll take a second look into it when I have some spare cycles, so keeping it as a draft.
But if anyone wants to pick this up, feel free to do so!